### PR TITLE
fix: re-stencil

### PR DESCRIPTION
**Pull request by artemchernyak-outreach**
## Jira ID

[DT-2831]

## Notes for your reviewers

[DT-2831]: https://outreach-io.atlassian.net/browse/DT-2831?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jaredallard — APPROVED** [review]

File: pkg/cli/updater/updater.go [hunk 1]
```diff
@@ -272,7 +272,7 @@ func (u *updater) check(ctx context.Context) (bool, error) {
 
 		// if we're not past the last update threshold
 		// then we don't check for updates.
-		if !time.Now().After(repoCache.LastChecked.Add(checkAmount)) {
+		if time.Now().Before(repoCache.LastChecked.Add(checkAmount)) {
```
Comment (jkinkead) [inline pkg/cli/updater/updater.go:275]: Can you revert this line? It's that horrible linter bug we fixed; just ignore the warning.

**Review by jkinkead — APPROVED** [review]
LGTM barring the bug callout! Make sure to fix.

File: pkg/cli/updater/updater.go (hunk 1 above)
Comment (artemchernyak-outreach) [reply pkg/cli/updater/updater.go:275]: Was this only fixed on RC? I am still seeing this error come up. I've added a no ling just in case.

Comment (jkinkead) [reply pkg/cli/updater/updater.go]: Possibly. I thought it had made it into the last release, but maybe not.

**Pull request merged**